src-vue/src/views/mods/LocalModsView.vue

```diff
@@ -5,44 +5,19 @@
     </div>
 
     <el-scrollbar v-else>
-        <el-card shadow="hover" v-for="mod in mods" v-bind:key="mod.name">
-            <el-switch style="--el-switch-on-color: #13ce66; --el-switch-off-color: #8957e5" v-model="mod.enabled"
-                       :before-change="() => updateWhichModsEnabled(mod)" :loading="global_load_indicator" />
-            <el-popconfirm
-                :title="$t('mods.local.delete_confirm')"
-                :confirm-button-text="$t('generic.yes')"
-                :cancel-button-text="$t('generic.no')"
-                @confirm="deleteMod(mod)"
-            >
-                <template #reference>
-                    <el-button type="danger">
-                        {{ $t('mods.local.delete') }}
-                    </el-button>
-                </template>
-            </el-popconfirm>
-            {{ mod.name }}
-            <span v-if="mod.version != null">(v{{ mod.version }})</span>
-            <img
-                v-if="mod.thunderstore_mod_string != null"
-                :title="$t('mods.local.part_of_ts_mod') + '\n' + mod.thunderstore_mod_string"
-                src="/src/assets/thunderstore-icon.png"
-                class="image"
-                height="16"
-            />
-        </el-card>
+    <local-mod-card v-for="mod of mods" v-bind:key="mod.name" :mod="mod" />
```

Comment:
Is this indented correctly?

Reply:
Fixed in 15753b6149e1b82c1a08de5a5c7fd13e5299d3f2